scene/gui/menu_button.cpp

```diff
@@ -60,7 +60,7 @@ void MenuButton::pressed() {
 	gp.y += get_size().y;
 
 	popup->set_position(gp);
-
+	popup->set_extra_height_limit(popup->get_position().y);
```

Comment:
If you put the MenuButton down really low on the screen, wouldn't it essentially make the popup miniscule and unusable?

Reply:
@EricEzaM We can make the menu go above the button in that case, just like what the auto-completion panel does in the script editor.

Reply:
@EricEzaM Or we leave `height_limit` as it is, and just make PopupMenu not respond to the mouse release too fast after the mouse press just activates it as it does in `3.2`.
![popup_menu_6](https://user-images.githubusercontent.com/28705694/109622449-1515b180-7b77-11eb-8461-83af9f690514.gif)